services/app/controllers/services_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ServicesController < ApplicationController
+    def index
+        @services = Service.all
+    end
+    
+    def show
+        @service = Service.find(params[:id])
+        @users =User.all
+    end
+    
+    def new
+        @service = Service.new
+    end
+    
+    def create
+        service = Service.new
```

Comment:
You are close! Your service form isn't saving a service. The reason why is because you are using a local variable `service`. Local variables are only available within their methods. In this case we want an instance variable so that it can be used outside of this method in your service create form. Try changing this to `@service` within this method that way your form knows what `@service` is.

[Here](https://learn.co/lessons/ruby-instance-variables) is a detailed blog explaining it in detail.